apprentice-react-o-shop/.gitignore

```diff
@@ -0,0 +1 @@
+/node_modules
```

Comment:
hmm it looks like everything is nester in an apprentice-react-o-shop. Move its contents up one directory and delete this folder